frontend/src/components/managePermissions/table/DelegatedAdminTable.vue

```diff
@@ -0,0 +1,126 @@
+<script setup lang="ts">
+import { ref, type PropType } from 'vue';
+import { FilterMatchMode } from 'primevue/api';
+import Column from 'primevue/column';
+import DataTable from 'primevue/datatable';
+
+import { routeItems } from '@/router/routeItem';
+import DataTableHeader from '@/components/managePermissions/table/DataTableHeader.vue';
+import type { FamAccessControlPrivilegeGetResponse } from 'fam-admin-mgmt-api/model';
+
+const props = defineProps({
+    loading: {
+        type: Boolean,
+        default: false,
+    },
+    delegatedAdmins: {
+        type: [Array] as PropType<FamAccessControlPrivilegeGetResponse[] | undefined>,
+        required: true,
+    },
+});
+
+const delegatedAdminFilters = ref({
+    global: { value: '', matchMode: FilterMatchMode.CONTAINS },
+    'user.user_name': { value: null, matchMode: FilterMatchMode.STARTS_WITH },
+    'user.user_type.description': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'role.role_name': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
+    'role.parent_role.role_name': {
+        value: null,
+        matchMode: FilterMatchMode.CONTAINS,
+    },
```

Comment:
the filter need to add the forest client number as well

Reply:
good catch! I'll add it